hw11.py

```diff
@@ -0,0 +1,48 @@
+### decorated function ###
+from datetime import datetime
+def decorator(my_func):
+    def deco_func(*args, **kwargs):
+        print(my_func.__name__)
+        print(datetime.now())
+        return my_func()

```

Comment:
pass *args and **kwargs here. We use these special variables to be able to decorate functions that take any parameters.